src/navigator/SettingsMenu.tsx

```diff
@@ -172,57 +150,36 @@ export default function SettingsMenu({ route }: Props) {
     dispatch(devModeTriggerClicked())
   }
 
+  const onCopyText = (...text: Array<string | null>) => {
+    return () => {
+      Logger.showMessage('Copied to Clipboard')
+      Clipboard.setString(text.join(', '))
+    }
+  }
+
   const getDevSettingsComp = () => {
     if (!devModeActive) {
       return null
     } else {
+      const statsigStableId = Statsig.getStableID()
       return (
         <View style={styles.devSettings}>
+          <Touchable onPress={onCopyText(sessionId)} style={styles.devSettingsItem}>
+            <Text>{`Session ID: ${sessionId}`}</Text>
+          </Touchable>
+          <Touchable onPress={onCopyText(statsigStableId)} style={styles.devSettingsItem}>
+            <Text>{`Statsig Stable ID: ${statsigStableId}`}</Text>
+          </Touchable>
           <View style={styles.devSettingsItem}>
-            <Text style={typeScale.labelSemiBoldSmall}>Session ID</Text>
-            <SessionId sessionId={sessionId || ''} />
-          </View>
-          <View style={styles.devSettingsItem}>
-            <TouchableOpacity onPress={toggleNumberVerified}>
-              <Text>Toggle verification done</Text>
-            </TouchableOpacity>
-          </View>
-          <View style={styles.devSettingsItem}>
-            <TouchableOpacity onPress={handleResetAppOpenedState}>
-              <Text>Reset app opened state</Text>
-            </TouchableOpacity>
-          </View>
-
-          <View style={styles.devSettingsItem}>
-            <TouchableOpacity onPress={handleToggleBackupState}>
-              <Text>Toggle backup state</Text>
-            </TouchableOpacity>
-          </View>
-          <View style={styles.devSettingsItem}>
-            <TouchableOpacity onPress={showDebugScreen}>
-              <Text>Show Debug Screen</Text>
-            </TouchableOpacity>
-          </View>
-          <View style={styles.devSettingsItem}>
-            <TouchableOpacity onPress={Sentry.nativeCrash}>
-              <Text>Trigger a crash</Text>
-            </TouchableOpacity>
-          </View>
-          <View style={styles.devSettingsItem}>
-            <TouchableOpacity onPress={wipeReduxStore}>
-              <Text>Wipe Redux Store</Text>
+            <TouchableOpacity onPress={showDebugImagesScreen}>
+              <Text>See app assets</Text>
```

Comment:
nit: to be consistent with the other action
```suggestion
              <Text>See App Assets</Text>
```